modules/setting/setting.go

```diff
@@ -25,6 +25,8 @@ import (
 	"github.com/gogits/gogs/modules/bindata"
 	"github.com/gogits/gogs/modules/log"
 	"github.com/gogits/gogs/modules/user"
+
+	"github.com/strk/go-libravatar"
```

Comment:
btw, does the "must be a URL from a code hosting site" wording in the go-code-convension url means hosting sites other than github and bitbucket are finally accepted ?